### ISSUE TITLE
Ergonomic pane layout: Position console pane on left side of source column.

### ISSUE THREAD

**Issue by canderson318** [rstudio/rstudio#14290]
Please allow placement of the console pane to the left side of the source column. "Global Options"> "Pane Layout" does not allow columns to be placed anywhere other than the left side. This way scripts are more easily read with lines starting in the middle of the user's screen. 

Thank you

**Comment by ronblum** [comment]
@canderson318 Thanks for raising this! Do you mean when there are multiple source panes?

**Comment by canderson318** [comment]
Not necessarily. I am referring to the source column on the left of the layout manager shown below. What I would like to do is move the source column to the right side, which would put the source code in the middle of my screen, i.e. a more ergonomic editing scheme.

Is that helpful?  
And thanks for the question! 

![Screenshot 2024-02-22 at 11 10 54](https://github.com/rstudio/rstudio/assets/125529390/e02e00fa-a49b-470a-b327-7cb0c9cf8c5e)

**Comment by ronblum** [comment]
@canderson318 Thanks! I'll mark this issue for review as part of our ongoing development.
